README.md

```diff
@@ -33,11 +33,11 @@ These steps come from the [extension dev page](https://developer.mozilla.org/en-
 
```

Comment:
I think we'll also need to update line 17 of this file, so that the example proxy file contents point to the src/ directory, not the addon root directory.

Which reminds me, are you game to remove the stuff about https (lines 24-26)? Now that the content server does https for us, setup is far simpler. Instead of setting up https, you just have to navigate to the local iframe's URL and accept the security exception.